api/client/batch_client.py

```diff
@@ -102,18 +104,20 @@ def get_df(self, show_revisions=True):
     # TODO: deprecate  the following  two methods, standardize  on one
     # approach with get_data_points and get_df
     @gen.coroutine
-    def get_data_points_generator(self, **selection):
+    def get_data_points_generator(self, include_historical=True, **selection):
         headers = {'authorization': 'Bearer ' + self.access_token}
         url = '/'.join(['https:', '', self.api_host, 'v2/data'])
         params = lib.get_data_call_params(**selection)
         resp = yield self.get_data(url, headers, params)
         raise gen.Return(json_decode(resp))
 
     def batch_async_get_data_points(self, batched_args, output_list=None,
-                                    map_result=None):
+                                    map_result=None, include_historical=True):
+        for args in batched_args:
```

Comment:
Can put the requirement on the caller to include it in each of the `batched_args` like

```py
batched_selections = [{
    'metric_id': 2100031,
    'item_id': 2039,
    'region_id': region_batch,
    'source_id': 35,
    'frequency_id': 1,
    'insert_null': True,
    'include_historical': False
} for region_batch in batch(batch_size)]
client.batch_async_get_data_points(batched_selections)
````

instead of calling `batch_async_get_data_points` with an additional arg. And this way if you really wanted to, it could vary from selection to selection - you might not want the same option on all of them

Reply:
discussed offline. previously had wanted to keep it separate from entities, but these will be removed in respective lib `get_data_call_params` fn and allows for batch call to have an option per selection. pushed changes to remove as an explicit option and keep as a selection